beacon_node/http_api/src/publish_blocks.rs

```diff
@@ -227,12 +227,16 @@ pub async fn publish_block<T: BeaconChainTypes, B: IntoGossipVerifiedBlockConten
         }
         Ok(AvailabilityProcessingStatus::MissingComponents(_, block_root)) => {
             let msg = format!("Missing parts of block with root {:?}", block_root);
-            error!(
-                log,
-                "Invalid block provided to HTTP API";
-                "reason" => &msg
-            );
-            Err(warp_utils::reject::broadcast_without_import(msg))
+            if let BroadcastValidation::Gossip = validation_level {
```

Comment:
Is a gossip error here always going to be a duplicate error? 🤔

Reply:
What do you mean by duplicate error?

Reply:
If you mean these two error, the first error is just if the publish itself fails, the second set of error handling is if the publish was successful

https://github.com/realbigsean/lighthouse/blob/more-deneb-cleanup/beacon_node/http_api/src/publish_blocks.rs#L138C31-L138C31

https://github.com/realbigsean/lighthouse/blob/more-deneb-cleanup/beacon_node/http_api/src/publish_blocks.rs#L231